packages/flutter/lib/src/material/input_border.dart

```diff
@@ -308,7 +308,7 @@ class OutlineInputBorder extends InputBorder {
     double gapPadding,
   }) {
     return new OutlineInputBorder(
-      borderSide: borderSide ?? this.borderSide,
+      borderSide: this.borderSide ?? borderSide,
```

Comment:
This doesn't seem right. It would mean that if you called copyWith, you could never override the borderSide.